apps/wallet/src/features/Home/index.tsx

```diff
@@ -45,8 +69,18 @@ const styles = StyleSheet.create({
 		alignItems: 'center',
 		gap: 36,
 	},
+	headerContainer: {
+		width: '100%',
```

Comment:
try to not use `width: '100%'`

Reply:
remove `alignItems: center` in the container `View`, then we don't need this `width: 100%`.
to align center something, in container level.. put that chunk of element under another wrapped `View` that itself do `alignItems: center`, or if there is only one single item needed to be center use `alignSelf: center` on it.